tools/validator-client-scripts/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "validator-client-scripts"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+base64 = "0.13.0"
+bs58 = "0.4"
+clap = { version = "3.2.8", features = ["derive"] }
+csv = "1.1"
+dotenv = "0.15.0"
+log = "0.4"
+pretty_env_logger = "0.4"
+serde = { version = "1", features = ["derive"] }
+serde_json = "1"
+tokio = { version = "1.11", features = [ "net", "rt-multi-thread", "macros", "signal"] }
+streaming-stats = "0.2.3"
+
+
+# we don't really need those directly, but including them somehow solves dependency issue
+url = "2.2"
```

Comment:
see if you can remove those now actually. What what I recall I had to include them because I was depending on `validator-client` through `git`